.authors.yml

```
@@ -888,3 +888,10 @@
   num_commits: 1
   first_commit: 2020-12-29 19:51:32
   github: timgates42
+- name: Thomas Donoghue
+  email: tdonoghue.research@gmail.com
+  alternate_emails:
+  - tdonoghue@ucsd.edu
```

```suggestion
  - tdonoghue@ucsd.edu
aliases:
  - Tom Donoghue
  - Tom
```

I think this is what it's asking for. I'll apply the suggestion now and see if it makes the CI happy.